pages/edit_group.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.page import PageRegion
+from pages.base import Base
+
+
+class Group(Base):
+    _description_button_locator = (By.ID, 'description-tab')
+    _description_tab_locator = (By.ID, 'description')
+    _access_button_locator = (By.ID, 'access-tab')
+    _access_tab_locator = (By.ID, 'access')
+    _invitations_button_locator = (By.ID, 'invitations-tab')
+    _invitations_tab_locator = (By.ID, 'invitations')
+
+    @property
+    def description(self):
+        self.selenium.find_element(*self._description_button_locator).click()
+        return self.DescriptionTab(self.base_url, self.selenium,
+                                   self.selenium.find_element(*self._description_tab_locator))
+
+    @property
+    def access(self):
+        self.selenium.find_element(*self._access_button_locator).click()
+        return self.AccessTab(self.base_url, self.selenium,
+                              self.selenium.find_element(*self._access_tab_locator))
+
+    @property
+    def invitations(self):
+        self.selenium.find_element(*self._invitations_button_locator).click()
+        return self.InvitationsTab(self.base_url, self.selenium,
+                                   self.selenium.find_element(*self._invitations_tab_locator))
+
+    class DescriptionTab(PageRegion):
+        _description_form_locator = (By.ID, 'description-form')
+        _delete_panel_locator = (By.CSS_SELECTOR, '.panel-danger')
+
+        @property
+        def description_info(self):
+            return self.DescriptionForm(self.base_url, self.selenium,
+                                        self._root_element.find_element(*self._description_form_locator))
+
+        @property
+        def delete_group(self):
+            return self.DeletePanel(self.base_url, self.selenium,
+                                    self._root_element.find_element(*self._delete_panel_locator))
+
+        class DescriptionForm(PageRegion):
```

Comment:
True, but I think it's better to use the same way of splitting these as we do all the Settings/Groups sections. Besides these could be expanded in the future.